### ISSUE TITLE
Colors are not reset in Windows when using Term::write

### ISSUE THREAD

**Issue by r-darwish** [mitsuhiko/console#69]
```rust
use console::{style, Color, Term};
use std::io::Write;

fn main() {
    let mut term = Term::stdout();
    term.write_fmt(format_args!(
        "{}",
        style("Grass green. I hate that color!\n").fg(Color::Green)
    ))
    .unwrap();
    term.write_line("Shouldn't be green");
}
```

The above program works as expected in Windows when used with console v0.10.1. However, with console v0.11.3 the second line is also green.

**Comment by r-darwish** [comment]
Bisection suggests that eb1fa993c7280113aff7222fcce87bb7f37551ea introduced the issue.

**Comment by pksunkara** [comment]
I will get to this in a few days. But this should be an easy fix if you are willing to make a PR.

**Comment by pksunkara** [comment]
Also, what's the window version you are using?

**Comment by r-darwish** [comment]
Window 10 version 2004

**Comment by pksunkara** [comment]
* Is the commit before that working?
* Do you have `windows-console-colors` feature enabled?
* What are the values of `Term::is_msys_tty` and `Term::is_tty`?

**Comment by mitsuhiko** [comment]
@pksunkara definitely broken now. Looking at the code now I wonder if this is not a bit too much hackery for getting this working on an effectively unsupported windows version.

**Comment by pksunkara** [comment]
This is fixed in 0.14.0 of console and will be propagated to dialoguer
